wire-ios/Wire-iOS/Sources/UserInterface/Calling/CallController/ActiveCallRouter.swift

```diff
@@ -61,7 +69,8 @@ final class ActiveCallRouter: NSObject {
 
     private var isCallQualityShown = false
     private var isCallTopOverlayShown = false
-    private(set) var scheduledPostCallAction: (() -> Void)?
+    private(set) var scheduledPostCallAction: PostCallAction?
+    private(set) var presentedDegradedAlert: UIAlertController?
```

Comment:
`UIViewController` should be `weak`, because the view hierarchy retains them and the weak var automatically becomes `nil`.

Reply:
good point! done